examples/unix_socket/dub.json

```diff
@@ -0,0 +1,11 @@
+{
+	"name": "unit-socket-example",
+	"description": "Example for sending http requests to unix sockets",
+	"dependencies": {
+		"vibe-d:http": {"path": "../../"},
+		"vibe-d:web": {"path": "../../"},
+		"vibe-d:core": {"path": "../../"}
+	},
+	"versions": ["VibeCustomMain", "VibeNoSSL"],
+	"subConfigurations": { "vibe-d:core": "libasync"}
```

Comment:
`"VibeCustomMain"` can be dropped here (it's the default now).

The libasync based driver is selected here, although the changes are for the libevent one - is this intended? Does the libasync driver work?